subsys/bluetooth/controller/ll_sw/ull_slave.c

```diff
@@ -99,6 +100,19 @@ void ull_slave_setup(memq_link_t *link, struct node_rx_hdr *rx,
 	win_offset = sys_le16_to_cpu(pdu_adv->connect_ind.win_offset);
 	conn_interval_us = interval * 1250U;
 
+	if (0) {
+#if defined(CONFIG_BT_CTLR_ADV_EXT)
+	} else if (adv->lll.aux) {
+		if (adv->lll.phy_s & BIT(2)) {
```

Comment:
Don't have some PHY macros that we could use instead of `BIT(2)`? I know there's `BT_HCI_LE_EXT_SCAN_PHY_CODED` but not sure if we want to use something with the `SCAN` infix here.

Reply:
We do, there are `PHY_xxx` macros but they are not quite used. I started separate change where it's updated everywhere in code so will address this in another PR.